build.gradle

```diff
@@ -330,6 +340,7 @@ dependencies {
     testImplementation 'org.springframework:spring-beans:5.3.20'
     testImplementation 'org.junit.jupiter:junit-jupiter:5.8.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    testImplementation "org.opensearch:common-utils:${common_utils_version}"
```

Comment:
What happens if common utils isn't available as we are attempting to create a fresh release?

Reply:
We have two options here: 
1. Remove the common-utils dependency
2. Hard-code the version number that we know is always going to be available. 
I've asked similar question here: https://github.com/opensearch-project/security/pull/1958#discussion_r929130618